src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/CuratedContentToolModal.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by wikia on 2015-08-19.
```

Comment:
As @bognix noticed, please remove that comment. Use @ownership or leave it empty